src/buckets/components/BucketCardMeta.tsx

```diff
@@ -55,6 +56,10 @@ const BucketCardMeta: FC<Props> = ({bucket, notify}) => {
     <span data-testid="bucket-schemaType"> {schemaLabel} </span>
   )
 
+  const bucketInfo = CLOUD
+    ? [schemaBlock, persistentBucketMeta]
```

Comment:
please reverse the order!  (to keep the current order of the meta data, then the schema block